guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -0,0 +1,43 @@
+[id="creating-a-webhook_{context}"]
+= Creating a Webhook
+
+.Procedure
+
+. In the {Project} web UI, click *Administer* > *Webhooks*.
+. Click *Create Webhook*.
+. Click *Subscribe to field* to reveal a list of available events.
+. Select an event from the list.
+. Enter a name.
+. Enter a target URL. When using the shellhooks plugin, the URL should be in the form of `https://{smartproxy.example.com}:{smartproxy_port}/shellhooks/my_script`.
+. Click *Template* to see the list of available templates.
+. Select a template from the list.
+. Enter a HTTP method.
+. Ensure that the *Enabled* flag is checked if you want to create active webhook.
+
+In the Credentials tab:
+
+. Optional: Enter the username and password when HTTP authentication is required. Do not enter anything when using shellhooks.
+. Select *Verify SSL* if the server certificate should be verified against the system certificate store or Foreman CA. When using shellhooks, ensure this flag is selected.
```

Comment:
Please use one sentence per line. As per our agreed style. This will not effect the formatting.